### DISCUSSION TITLE
Question about code submission

### DISCUSSION THREAD

**Discussion by hansu1017** [PaddlePaddle/PaddleSpatial#293] [BaiduKDDCup2022]
Hi, Host,

We uploaded the source code but got an upload error, what's the detailed reason of this error?

Looking forward to your reply.

**Comment by shaido987** [comment]
Yes, I also tried it after seeing your question here and it does not work for me either. Since the email did not mention the need to upload the source code on the competition webpage I assume that this is not needed. Most likely it is enough to provide a link to the code in the abstract of the technical report (uploaded to e.g. github).

@ramber1836 @zhoujingbo Please correct me if I'm wrong here and I will try to upload the code again. :)

**Reply by ramber1836** [reply]
Yes, the link to the source code in the technical report is required.
